app/presenters/indexable_presenter.rb

```diff
@@ -0,0 +1,25 @@
+class IndexablePresenter
+
+  def initialize(policy)
+    @policy = policy
+  end
+
+  def indexable_attributes
+    {
+      title: policy.name,
+      description: policy.description,
+      link: id,
+      indexable_content: "",
```

Comment:
The description is already indexed. This field would normally be the body of a document or something similar.